sdk/core/core-amqp/CHANGELOG.md

```diff
@@ -41,12 +35,6 @@
 
 - Address React-Native regression after ESM migration [Issue #30065](https://github.com/Azure/azure-sdk-for-js/issues/30065)
 
-## 4.4.0-beta.1 (2024-06-06)
```

Comment:
why removing this entry?

Reply:
I ran the script and not sure why it was removed.

Reply:
Restored